cluster/slave/api_backend.go

```diff
@@ -425,14 +425,14 @@ func (s *SlaveBackend) getMinorBlockHeadersWithSkip(gReq *p2p.GetMinorBlockHeade
 		mTip   = shrd.MinorBlockChain.CurrentBlock()
 	)
 	if gReq.Type == qcom.SkipHash {
-		iHeader := shrd.MinorBlockChain.GetHeader(gReq.GetHash())
+		iHeader := shrd.MinorBlockChain.GetBlock(gReq.GetHash())
```

Comment:
iBlock := shrd.MinorBlockChain.GetBlock(gReq.GetHash())
		if qcom.IsNil(iBlock) {
			return nil, fmt.Errorf("failed to get minor block header by hash, minor hash: %s", gReq.GetHash().String())
		}
		height = iBlock.NumberU64()
		hash, err := shrd.MinorBlockChain.GetHashByHeight(&height)
		if err != nil || hash != iBlock.Hash() {
			return nil, fmt.Errorf("failed to get minor block header by number or hash dont match, minor number: %d", height)
		}